assets/prototype/domain/shared/predicates/shared/selectionPredicates/index.ts

```diff
@@ -17,7 +17,7 @@ export const entityHasDbId = (dbId: EntityDbId) => propEq('dbId', dbId);
 export const entityHasGuid = (guid: EntityId) => propEq('id', guid);
 
 // returns entity if found in array of entities
-export const findEntityByDbId = (entities: Entity[]) => (dbid: EntityDbId): Entity => {
+export const findEntityByDbId = (entities: Entity[]) => (dbid: EntityDbId): Entity | Price => {
 	return find(entityHasDbId(dbid))(entities);
 };
```

Comment:
errrr... don't think this shared predicate should be so specific.
did you **need** to add this? The `Price` type _should_ be extending `Entity` and should therefore not interfere with typings for predicates that use this for filtering `Prices`.

If it is not working that way, then we should likely do something like:

```ts
export const findEntityByDbId = <T extends Entity>(entities: T[]) => (dbid: EntityDbId): T => {
```

which uses a generic so that consuming logic can specify the exact type. 
The generic is extending the `Entity` interface though so it won't allow types that do not satisfy that interface and therefore guarantee that the supplied object will have a `dbId` property.

Reply:
I tried to tame these funcs: 
![image](https://user-images.githubusercontent.com/1477453/73196470-4c29c000-4138-11ea-9b04-bbed5a53afe5.png)

Reply:
Thx for the suggestion, works perfectly: https://github.com/eventespresso/event-espresso-core/pull/2286/commits/2462a5282fbacaeefd25d922a8972bb3b6446110

:bowing_man: